docs/system_setup.rst

```diff
@@ -0,0 +1,390 @@
+.. code:: shell
+
+    wget https://raw.githubusercontent.com/eWaterCycle/ewatercycle/master/environment.yml
+
+.. code:: shell
+
+    conda env create --file environment.yml
+
+.. code:: shell
+
+    conda activate ewatercycle
+
+Do not forget that any terminal or Jupyter kernel should activate the conda environment before the ewatercycle Python package can be used.
+
+Install ewatercycle package
+---------------------------
+
+The Python package can be installed using pip
+
+.. code:: shell
+
+    # TODO once released install from PyPi instead of GitHub
+    # pip install ewatercycle
+    pip install git+https://github.com/eWaterCycle/ewatercycle.git#egg=ewatercycle
+
+
+Configure ESMValTool
+--------------------
+
+ESMValTool is used to generate forcing (temperature, precipitation,
+etc.) files from climate data sets for hydrological models. The
+ESMValTool has been installed as a dependency of the package.
+
+See https://docs.esmvaltool.org/en/latest/quickstart/configuration.html
+how configure ESMValTool
+
+Download climate data
+---------------------
+
+The ERA5 and ERA-Interim climate datasets can be used to generate
+forcings
+
+ERA5
+~~~~
+
+To download ERA5 data files you can use the
+`era5cli <https://era5cli.readthedocs.io/>`__ tool.
+
+.. code:: shell
+
+    pip install era5cli
+
+To run the example notebooks the hourly ERA5 data files for year 1990
+and 1991 and for variables: pr, psl, tas, taxmin, tasmax, tdps, uas,
+vas, rsds, rsdt and fx orog need to be downloaded with
+
+.. code:: shell
+
+    # TODO is era5cli command is correct?
+    cd <ESMValTool raw directory>
+    era5cli hourly --startyear 1990 --endyear 1991 --variables pr psl tas taxmin tasmax tdps uas vas rsds rsdt orog
+    cd -
+
+.. code:: shell
+
+    # TODO cmorize files?
+
+ERA-Interim
+~~~~~~~~~~~
+
+ERA-Interim has been superseeded by ERA5, but could be useful for
+reproduction studies and its smaller size. The ERA-Interim data files
+can be downloaded at
+https://www.ecmwf.int/en/forecasts/datasets/reanalysis-datasets/era-interim
+
+Installing a container engine requires root permission on the machine.
+
+Install container engine
+------------------------
+
+In ewatercycle package the hydrological models are run in containers
+with engines like `Singularity <https://singularity.lbl.gov/>`__ or
+`Docker <https://www.docker.com/>`__. At least Singularity or Docker
+should be installed.
+
+Singularity
+~~~~~~~~~~~
+
+Install Singularity using
+`instructions <https://singularity.hpcng.org/user-docs/master/quick_start.html>`__.
+
+Docker
+~~~~~~
+
+Install Docker using
+`instructions <https://docs.docker.com/engine/install/>`__. Docker
+should be configured so it can be `called without
+sudo <https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user>`__
+
+Configure ewatercycle
+---------------------
+
+The ewatercycle package simplifies the API by reading some of the
+directories and other configurations from a configuration file.
```

Comment:
```suggestion
The eWaterCycle package simplifies the API by reading some of the
directories and settings from a configuration file.
```

Reply:
Maybe elaborate a bit more on "the API" here